antora.yml

```diff
@@ -35,6 +35,7 @@ asciidoc:
       https://github.com/serverlessworkflow/specification/blob/0.8.x/specification.md
```

Comment:
We need to update the downstream versions here.

Reply:
`quarkus_version`: 2.13.5.Final-redhat-00003
`kogito_version_redhat`: 1.32.0.Final-redhat-00003
`kn_cli_version`: 0.26.0